storehaus-algebra/src/main/scala/com/twitter/storehaus/algebra/MergeableStore.scala

```diff
@@ -31,25 +32,29 @@ object MergeableStore {
   implicit def enrich[K, V](store: MergeableStore[K, V]): EnrichedMergeableStore[K, V] =
     new EnrichedMergeableStore(store)
 
+  private[this] def addOpt[V](init: Option[V], inc: V)(implicit sg: Semigroup[V]): Option[V] = init match {
```

Comment:
1. the latest algebird has `Semigroup.maybePlus` this does this, but it always returns `V` not `Option[V]` which I think is correct (you can just box in `Some` as needed).